tox.ini

```diff
@@ -8,7 +8,9 @@ usedevelop = true
 extras = dev
 deps =
     nameko{2.6,2.7}: pytest<3.3.0
-    nameko{2.6,2.7}: eventlet<0.22.0
+    nameko{2.6,2.7,2.8}: eventlet<0.22.0
+    nameko{2.6,2.7,2.8,2.9.2.10}: kombu>=3.0.1,<4
```

Comment:
I don't think we need this. All these versions of `nameko` already pin `kombo` to these versions.